frontend/src/metabase/dashboard/components/DashCard/DashCardParameterMapper/DashCardCardParameterMapper.unit.spec.jsx

```diff
@@ -41,20 +42,20 @@ const metadata = getMetadata(state); // metabase-lib Metadata instance
 const setup = options => {
   const card = options.card ?? createMockCard();
 
-  renderWithProviders(
+  const { rerender } = renderWithProviders(
     <DashCardCardParameterMapper
       card={card}
       dashcard={createMockDashboardCard({ card })}
       question={new Question(card, metadata)}
       editingParameter={createMockParameter()}
-      isRecentlyAutoConnected={options.isRecentlyAutoConnected ?? false}
+      isRecentlyAutoConnected={false}
       mappingOptions={[]}
-      metadata={metadata}
```

Comment:
`metadata` and `setParameterMapping` are not passed anymore, but file is not TS and it's not easy to convert it to TS, so I'm just removing props manually